### PR TITLE
153-blognews-back-social-media-banner

### LINKED ISSUE #153: [Blog/News] Implement back & social media banner
Implement the back link & social media banner via the page template autoblocking. This is based on #151

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Fix #153 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/how-cell-painting-making-mark
- After: https://153-blognews-back-social-media-banner--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/how-cell-painting-making-mark

### CODE REVIEW

File: templates/blog/blog.js [hunk 1]
```diff
@@ -1,5 +1,28 @@
 import { buildBlock } from '../../scripts/lib-franklin.js';
 
+const social = `<div class="social">
```
Comment (mhaack) [inline templates/blog/blog.js:3]: Is `<div class="social">` nedded? I think we can maybe skip that extra div as the block code already adds a div.

**Comment by sharanagoudapatil** [comment]
image

@mhaack - Please verify the changes. Thanks.

File: blocks/social-media/social-media.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.social {
+    @apply relative z-10 flex items-center justify-between mt-4 mb-2 text-gray-600;
+}
+
+.back-btn{
```
Comment (mhaack) [inline blocks/social-media/social-media.css:5]: should we scope these with `.social .back-btn`?

**Comment by mhaack** [comment]
Screenshot 2023-10-10 at 13 37 44
There are still 2 unnecessary div's we can avoid.

**Comment by sharanagoudapatil** [comment]
@mhaack - The above divs gets added from the buildBlock() function from lib-frankiln.js. 

image

File: blocks/social-media/social-media.css (hunk 2 above)
Comment (sharanagoudapatil) [reply blocks/social-media/social-media.css:5]: Thanks for this I've changed. Please review. Thanks.

**Comment by mhaack** [comment]
> @mhaack - The above divs gets added from the buildBlock() function from lib-frankiln.js.

Yes I know and we can used them `<div class="relative z-10 flex items-center justify-between mt-4 mb-2 text-gray-600 social">` could be avoided and you can try to use ` buildBlock('social-media', { elems: social }),` (without the array)

**Review by mhaack — APPROVED** [review]

**Pull request merged**